FlaskApp/app/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 #----DB SETTINGS BEGIN---------------------------------------------
 
-MONGO_HOST = 'localhost'
+MONGO_HOST = '162.243.122.37'
```

Comment:
should actually be imported from env. this will breake when you pull to your local host.